src/python/bot/untrusted_runner/host.py

```diff
@@ -160,7 +159,7 @@ def wrapped(*args, **kwargs):
           # TODO(ochang): Replace with generic TimeoutError in Python 3.
           # Use __str__ to avoid newstrs. TODO(ochang): Use plain str() once
           # migrated to Python 3.
-          raise engine.TimeoutError(e.__str__())
+          raise TimeoutError(e.__str__())
```

Comment:
can you also replace the `__str__` with a str() call and remove the comment above?

Reply:
oh, sure. Didn't read the comment above.